src/devices/sunricher.ts

```diff
@@ -56,6 +68,23 @@ async function syncTime(endpoint: Zh.Endpoint) {
 }
 
 const definitions: Definition[] = [
+    {
+        zigbeeModel: ['HK-SENSOR-4IN1-A'],
+        model: 'HK-SENSOR-4IN1-A',
+        vendor: 'Sunricher',
+        description: '4IN1 Sensor',
+        fromZigbee: [fz.ias_occupancy_alarm_1, fz.battery, fz.temperature, fz.humidity, fz.illuminance],
```

Comment:
`fromZigbee`, `toZigbee` and `deviceEndpoints({'endpoints': {'1': 1, '2': 2, '3': 3, '4': 4, '5': 5}}),` can be safely removed, please try if it works after that.

Reply:
Hello, thanks for your advice, the device works fine after that.